packages/terra-dropdown-button/src/Button.module.scss

```
@@ -0,0 +1,31 @@
+:local {
+  .button {
+    color: var(--terra-dropdown-button-menu-item-color, #1c1f21);
+    font-size: var(--terra-dropdown-button-menu-item-font-size, 0.9285714286rem);
+    line-height: var(--terra-dropdown-button-menu-item-line-height, 1.2);
+    outline: 0;
+    padding-bottom: var(--terra-dropdown-button-menu-item-padding-bottom, 4px);
+    padding-left: var(--terra-dropdown-button-menu-item-padding-left, 10px);
+    padding-right: var(--terra-dropdown-button-menu-item-padding-right, 10px);
+    padding-top: var(--terra-dropdown-button-menu-item-padding-top, 4px);
+
+    &.focus {
+      background: var(--terra-dropdown-button-menu-item-focus-background, #ebf6fd);
```

Let's change this to background-color.
```suggestion
      background-color: var(--terra-dropdown-button-menu-item-focus-background-color, #ebf6fd);
```